dev/dev_setup.sh

```diff
@@ -66,17 +70,14 @@ supported_platform() {
         Linux)
```

Comment:
You might want to distinguish between Debian-based and redhat-based distros here, since apt-get is Debian-only...

Reply:
ooh, nice. Fixed.